webview/cocoa.py

```diff
@@ -166,7 +201,18 @@ def webView_didFinishLoadForFrame_(self, webview, frame):
             if not webview.window():
                 BrowserView.instance.window.setContentView_(webview)
                 BrowserView.instance.window.makeFirstResponder_(webview)
-                
+
+                frame_size = BrowserView.instance.window.frame().size
+                drag_bar_height = 24
+
+                # Flip the webview so our bar is position from the top, not bottom
+                webview.setFlipped_(True)
+
+                rect = AppKit.NSMakeRect(0, 0, frame_size.width, drag_bar_height)
+                drag_bar = DragBar.alloc().initWithFrame_(rect)
+                drag_bar.setAutoresizingMask_(AppKit.NSViewWidthSizable)
+                BrowserView.instance.window.contentView().addSubview_(drag_bar)
+
```

Comment:
Is there a reason this code (line 204 to 215) is put here? Do we need to update the drag bar
every time the page loads? If not, the best place for it would be in the `__init__()` function
itself.

Reply:
I managed to move it to init in https://github.com/r0x0r/pywebview/pull/177/commits/841ca88e5f0ab76002ef9ca7f9e16d36d75b2b80 (new to pyobjc!).